src/components/ResolveDuplicateMRButtonAndModal/ResolveDuplicateMRButtonAndModal.js

```diff
@@ -243,11 +262,9 @@ const ResolveDuplicateMRButtonAndModal = ({
                 <IconPen /> {editEither}
               </ButtonCaution>
             </Thead>
-            <Thead>
+            <Thead aria-label="Duplicate Management">
```

Comment:
This is good enough for now, but in future tests, we should try to use selectors like `getByRole('columnheader', {name: 'bla bla bla'})`. Here this aria-label is unnecessary (to my knowledge) for a screen reader because the screen reader can tell its a column header (th) and what the text within is. In general using aria-label to help with tests is better than using test-id, but using other selectors is better than either of those unless aria label actually helps a screen reader that has no other way of interpreting the markup. 

Often in those cases, if time allows, its good to talk to Al about it, because those times are usually pointing out where the UX could use some explicit messaging (eg icon buttons)

But lets move on. We can work on that in the next set of tests. 

This is very good for being newish to authoring completely from scratch a bunch of tests in this very complicated application

Reply:
ah, ya, then my advice is cheap. Ill try pulling it and seeing. Feel free to ignore and merge if you would rather.

Reply:
K, so I figured out waht the issue is with selecting the table headers. 

This will work:
```
await within(resolveModal).findByRole('columnheader', {
    name: 'foo',
  })
  ```
  
  for 
  `<th scope="col">foo </th>` or `<th role="columnheader">foo</th>`
  
  
  But it will not work if there are other things nested in the `th` like buttons, which is what is going on with the resolve duplicates table.
  
  ```
  <th scope="col">
              foo <button>other stuff</button>
    </th>
            ```
            
            So `aria-label` is the right call for the ths that also have other stuff in them :)